message/model.json

```diff
@@ -1229,12 +1346,68 @@
                         "path": {
                           "name": "path",
                           "description": "The HTTP path as a uri template",
-                          "type": "string"
+                          "type": "uritemplate",
+                          "required": false
                         },
                         "method": {
                           "name": "method",
                           "description": "The HTTP method",
-                          "type": "string"
+                          "type": "string",
+                          "required": false
+                        }
+                      }
+                    }
+                  }
+                },
+                "NATS": {
+                  "siblingattributes": {
+                    "protocoloptions": {
+                      "name": "protocoloptions",
+                      "description": "NATS message metadata constraints",
+                      "type": "object",
```

Comment:
need: charset:extended

Reply:
There are no fields in NATS that require it

Reply:
"reply-to"